.dockerignore

```diff
@@ -12,7 +12,7 @@ persistence/
 scilla/
 scilla_files/
 config.xml
-isolated-server-accounts.json
+# isolated-server-accounts.json
```

Comment:
Comment why this is commented out?

Reply:
I needed to comment this line to be able to copy the isolated-server-accounts.json inside the Dockerfile to run the tests, otherwise it is ignored and not copied